src/legacy/core_plugins/kibana/public/table_list_view/table_list_view.js

```diff
@@ -62,8 +62,8 @@ class TableListViewUi extends React.Component {
       showLimitError: false,
       filter: this.props.initialFilter,
       selectedIds: [],
-      sortField: 'title',
-      sortDirection: 'asc',
+      sortField: '',
```

Comment:
Sorry I think I misunderstood you earlier. We cannot have the initial ordering (without querying) not by title (that was another issue, that was fixed just recently :D).

So basically we need a behavior: Without search: sort alphabetically by title, once searched sort by natural order (and I am fine if as soon as you click then a sorting you cannot return to natural order for now). I tried to test this locally and it seems like that behavior is really not achievable with `EuiInMemoryTable` right now, so it looks like we need to make those `EuiInMemoryTable` changes before we can continue with this.

Maybe we remove this from the PR right now, and only have it for the Button -> Link converation, and try to get the EUI changes in soon.